sunset/code/modules/admin/verbs/randomverbs.dm

```diff
@@ -0,0 +1,33 @@
+/obj/effect/overlay/adminfrozen
+	name = "FROZEN"
+	icon = 'sunset/icons/effects/effects.dmi'
+	icon_state = "adminfreeze"
+/
```

Comment:
What is with the random slash?

Reply:
it is the result of 3.5 million years of evolution 


and a typo

Reply:
fixed